Movies-MVP/Movies-MVP/Core/KeychainService/KeychainService.swift

```diff
@@ -0,0 +1,46 @@
+// KeychainService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import KeychainSwift
+
+/// Storage for critical data
+final class KeychainService {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let apiKeyValue = "API_KEY"
+        static let apiKey = "apiKey"
+    }
+
+    // MARK: - Public properties
+
+    static let shared = KeychainService()
+
+    // MARK: - Private properties
+
+    private let keyChain = KeychainSwift()
+
+    // MARK: - Initializer
+
+    private init() {}
+
+    // MARK: - Public methods
+
+    func readApiKey() -> String {
```

Comment:
```suggestion
    func readApiKey() -> String {
```
Глянь у Даниила эту реализацию там универсальней